ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -2093,7 +2093,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
+				CODE_SIGN_ENTITLEMENTS = BetterSocialStaging/BetterSocialStagingRelease.entitlements;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `CODE_SIGN_ENTITLEMENTS` path from `BetterSocial/BetterSocial.entitlements` to `BetterSocialStaging/BetterSocialStagingRelease.entitlements` seems to be for different build configurations. Ensure that the new entitlement file exists and contains the correct settings for the staging release configuration.
```diff
- CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
+ CODE_SIGN_ENTITLEMENTS = BetterSocialStaging/BetterSocialStagingRelease.entitlements;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->